{"name": "Demarrage", "status": "failed", "statusDetails": {"message": "Setup failed:\nMaxRetryError: HTTPConnectionPool(host='127.0.0.1', port=4723): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000024FE3E5D190>: Failed to establish a new connection: [WinError 10061] Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée'))\n\nAlso teardown failed:\nNo application is open"}, "start": 1689267291662, "stop": 1689267302590, "uuid": "fe31218f-6724-4088-ab43-f7fe242ff6be", "historyId": "84a88efa2949dbb22117d72f5b94008a", "fullName": "Tests.homeTest.Demarrage", "labels": [{"name": "suite", "value": "Tests"}, {"name": "subSuite", "value": "homeTest"}, {"name": "framework", "value": "robotframework"}, {"name": "language", "value": "cpython3"}, {"name": "host", "value": "LAPTOP-KC6KER4Q"}, {"name": "thread", "value": "default-99160b78-a014-4a38-980b-f043ce176506"}, {"name": "tag", "value": "test_001"}]}